dspace-server-webapp/src/main/java/org/dspace/app/rest/repository/ProcessRestRepository.java

```diff
@@ -0,0 +1,86 @@
+/**
+ * The contents of this file are subject to the license and copyright
+ * detailed in the LICENSE and NOTICE files at the root of the source
+ * tree and available online at
+ *
+ * http://www.dspace.org/license/
+ */
+package org.dspace.app.rest.repository;
+
+import java.sql.SQLException;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.dspace.app.rest.converter.processes.ProcessConverter;
+import org.dspace.app.rest.model.ProcessRest;
+import org.dspace.authorize.AuthorizeException;
+import org.dspace.authorize.service.AuthorizeService;
+import org.dspace.content.Process;
+import org.dspace.content.service.ProcessService;
+import org.dspace.core.Context;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.rest.webmvc.ResourceNotFoundException;
+import org.springframework.stereotype.Component;
+
+/**
+ * The repository for the Process workload
+ */
+@Component(ProcessRest.CATEGORY + "." + ProcessRest.NAME)
+public class ProcessRestRepository extends AbstractDSpaceRestRepository {
```

Comment:
Why isn't this extending `DSpaceRestRepository`?  It seems like we are not following our current development practices for these endpoints

Reply:
Please refer to the previous comment: https://github.com/DSpace/DSpace/pull/2529#discussion_r336980015